src/web/hvor_er_html/hvor_er_html.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "Hvor er HTML? Jeg ser den ikke!"
+level: 1
+language: nb-NO
+---
+
+# Introduksjon {.intro}
+
+Målet med denne oppgaven er å _vise_ hvor HTML finnes i virkelige nettsider, uten først å måtte pugge navnet på tags. Vi skal prøve å gi en intuitiv forståelse av HTML som trestruktur, uten alt for mye distraksjoner.
+
+Denne oppgaven er demonstrert med nettleseren Google Chrome. Bruk Google Chrome hvis du har den.
+
+# Steg 1: Se {.activity}
+
+Hvor er da all HTML-koden bak nettsidene vi ser til vanlig? Den er gjemt bakk nettleseren! La oss ta en titt.
+
+## Aktiviteter {.check}
+
+- Gå til [kodeklubben.github.io](http://kodeklubben.github.io) i Chrome eller Firefox
+- Høyreklikk på katten og trykk `Inspisér` (norsk) eller `Inspect` (engelsk).
+
+  ![](inspiser_katten.png)
+
+  Når vi klikker på `Inspiser`, får vi se en meny til hjelp for utviklere. Denne er svært nyttig til webutvikling.
+
+  ![](kattens_kildekode.png)
+
+  Ser du den fargede teksten, du også? Det er HTML, slik Chrome leser den! Nettsidens kildekode.
+
+  Kildekoden til katten ser slik ut:
+
+  ```html
+  <img src="scratch/logo-black.png">
+  ```
+
+- Se om du finner igjen `scratch/logo_black.png` i kildekoden. Hold pekeren over filnavnet. Ser du filen dukker opp?
+
+- Inspiser slangen. Hva er filnavnet til slangen? Ser du det samme som meg?
+
+  ![](slangens_kildekode.png)
+
+  Kildekoden til slangen ser slik ut:
+
+  ```html
+  <img src="python/logo-black.png">
+  ```
+
+## Hva har vi lært? {.protip}
+
+**Høyreklikk og inspect** finner HTML-koden til elementet vi ser på.
+
+**HTML-kode for bilder** ser slik ut:
+```html
+<img src="fil.png">
+```
+
+# Steg 2: Fjern {.activity}
+
+Nettleseren lar og tulle med HTML-koden til alle nettsider vi er på. Vi ødelegger ikke nettsidene, altså! Når vi gjør noe med nettsidene, er det bare i vår egen nettleser det skjer noe. Alle andre som ser på nettsiden ser en *helt vanlig* nettside.
+
+## Alt borte vekk? {.protip}
+
+Tips: Skulle du fjerne alt for mye, last siden på nytt med oppdater-knappen. Da blir alt som det var. Prøv!
+
+## Aktiviteter {.check}
+
+Jeg er glad i Minecraft. Men Minecraft er ikke først! La oss gjøre noe med det. Vi fjerner alt som er foran!
```

Comment:
Her synes jeg det står fint med to setninger. Korte setninger er lette å lese. Setninger bør ikke starte med "Men", men bryter den her for å få lettleselig språk.